app/schemas.py

```
@@ -867,4 +868,86 @@ class ImageOptimizationResponse(BaseModel):
+    base_currency: str = Field(default="USD", max_length=3)
+    target_currency: str = Field(default="BRL", max_length=3)
+    rate: Decimal = Field(..., gt=0, description="Exchange rate value")
+    source: Optional[str] = Field(None, max_length=50)
+    valid_from: date = Field(..., description="Start date of rate validity")
+    valid_to: date = Field(..., description="End date of rate validity")
+    organization_id: str = Field(..., description="UUID of organization")
+    
+    @validator('year_month')
+    def validate_year_month_format(cls, v):
+        if not re.method(r'^\d{4}-\d{2}$', v):
+            raise ValueError('year_month must be in format YYYY-MM')
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)